gnomad/utils/vcf.py

```diff
@@ -328,6 +328,42 @@
 Dictionary with in silico score descriptions to include in the VCF INFO header.
 """
 
+
+VRS_FIELDS_DICT = {
+    "VRS_Allele_IDs": {
+        "Number": "1",
```

Comment:
```suggestion
        "Number": "R",
```

Reply:
Are you referring to this? https://samtools.github.io/hts-specs/VCFv4.2.pdf
For example, if the INFO field contains a single number, then this value should be 1; if the INFO field describes a pair of numbers, then this value should be 2 and so on. There are also certain special characters used to define special cases:
• If the field has one value per alternate allele then this value should be ‘A’.
• If the field has one value for each possible allele (including the reference), then this value should be ‘R’.
• If the field has one value for each possible genotype (more relevant to the FORMAT tags) then this value
should be ‘G’.
• If the number of possible values varies, is unknown, or is unbounded, then this value should be ‘.’.

Reply:
I got the point now, R = "respectively"?